Resources/doc/reference/custom_fragment.rst

```diff
@@ -67,6 +78,7 @@ Using the twig helper, you will be able to access the following variables inside
 
 * ``fragment`` : The full fragment object.
 * ``fields`` : The values that were set in fragment settings.
+* ``foo`` : Any keys you may have defined in the ``getExtraContent`` method of the fragment.
```

Comment:
Please replace "in the ``getExtraContent`` method of the fragment" with " when implementing <whatever the interface name we decide to use>" so that people don't forget to write `implements TheInterface` in their code.